src/lib/components/dashboard/groups/AddUser.svelte

```diff
@@ -13,24 +16,41 @@
         await createUser(payload);
         showAddUserDrawer.set(false);
     };
+
+    const handleClose = () => {
```

Comment:
please use the store to close on the other file.

Reply:
done